docs/07_tools/02_eosio-helm/index.md

```diff
@@ -0,0 +1,87 @@
+---
+content_title: EOSIO Nodeos Helm Charts
+link_text: EOSIO Nodeos Helm Charts
+---
+
+## Summary
+
+* [Overview](#overview)
+* [Installation](#installation)
+    * [Local Deployment](#local-deployment)
+    * [Cloud Deployment](#cloud-deployment)
+* [Configuration](#configuration)
+    * [Customizing Values](#customizing-values)
+    * [Configuring Cloud Provider](#configuring-cloud-provider)
+* [Tutorial](#tutorial)
+
+## Overview
+
+EOSIO Helm Charts provide direct access to infrastructure-as-code (IaC) for rapid deployment of EOSIO nodes via Kubernetes. It includes support for common cloud providers such as Amazon Web Services (AWS) and Google Cloud Platform (GCP). EOSIO Helm Charts currently includes three charts:
```

Comment:
I think "EOSIO Helm Charts" here is a proper name, singular not a plural, a tool. It is proven by the next sentence which says "It includes" not "they include".
if that is true then `provides` not provide is in order.